src/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -146,18 +141,12 @@ const SavedAddresses: React.FC<SavedAddressesProps> = props => {
     setAddress(address)
   }
 
-  const createOrUpdateAddressSuccess = (
-    address?: UpdateUserAddressMutation$data & CreateUserAddressMutation$data
-  ) => {
-    refetchAddresses(() => {
-      if (address?.createUserAddress) {
-        onAddressCreate && onAddressCreate(address.createUserAddress)
-      } else if (address?.updateUserAddress) {
-        onAddressEdit && onAddressEdit(address.updateUserAddress)
-      }
-    })
-
-    onShowToast && onShowToast(true, "Saved")
+  const onSuccess = (id: string) => {
```

Comment:
@AlicanAkyuz I re-added the onAddressCreate/Edit functions from `index/shipping` as I realized that they include some additional logic for ARTA.

Reply:
I also changed them to accept the address ID instead of the entire address as we only pass the id to `onSucess()` in the form component

Reply:
makes sense! however, I think we need the full address passed in case of editing (or else we should look into refetching addresses in `index/shipping` after an address is edited here).

Reply:
Yep! I needed to update this to pass the address for your previous bug fix